### PR TITLE
Stylelint indentation

### PR DESCRIPTION
**Pull request by garrett**
Follow-up to https://github.com/cockpit-project/cockpit/pull/17408; the commits are the same except for the indentation one at the very end.

### CODE REVIEW

File: pkg/lib/cockpit-components-plot.scss [hunk 1]
```diff
@@ -1,64 +1,64 @@
...
-    black-500  #4d5258
+  blue-300   #06c,
+  green-100  #bde2b9,
+  cyan-200   #73c5c5,
+  purple-100 #b2b0ea,
+  gold-300   #f4c145,
+  orange-300 #ec7a08,
+  red-200    #a30000,
+  cyan-300   #009596,
+  black-500  #4d5258
```
Comment (garrett) [inline pkg/lib/cockpit-components-plot.scss:12]: Note: This is in a SCSS file and is correct for SCSS. Glancing over the commit made me wonder where the `:` are.. but this is right. :wink: It's a [SASS map](https://sass-lang.com/documentation/values/maps) with key value pairs.

**Comment by garrett** [comment]
Originally, I had these using spaces for indentation. (I forget if was 2 or 4.) After reading https://www.reddit.com/r/javascript/comments/c8drjo/nobody_talks_about_the_real_reason_to_use_tabs/, I set it to tabs (for accessibility reasons) when redoing this branch with current CSS.

**Comment by garrett** [comment]
When running the following command, it should only show the indentation configuration file change, as everything else is "whitespace":

```
git diff origin/main -w
```

(And, indeed, that's what I see locally.)

**Comment by marusak** [comment]
Can you please also add the enforcing rule for stylelint?

**Comment by garrett** [comment]
@marusak: I did?

Oh, I forgot to commit that this one time. Whoops. Fixing now.

**Comment by garrett** [comment]
I reset to main, re-added the indentation rule to our stylelint file (whoops), and re-ran stylelint with `--fix` to indent the files again... and force pushed the results.

**Comment by martinpitt** [comment]
[two pixel failures](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17510-20220811-182835-1fec26c6-fedora-36/log.html#62), retrying to compare. The second one is a flake.

**Comment by martinpitt** [comment]
Failed again.. but oh, your PR indeed updates pixel references.. I don't think that was intended. This merely updates whitespace/indentation, nothing functional. I assume this was an accident, so I'll re-push this without the pixel ref changes.

**Review by martinpitt — APPROVED** [review]
Ah, better.. that's how I like my PRs, nice and green! :green_heart: Thanks @garrett !

**Pull request merged**